README.md

```diff
@@ -51,24 +51,116 @@ If BSG decide to create the ability to do this on live OR I receive a DCMA reque
+* __**NOTE:**__ The SIT Client Mod should **NOT** be installed onto the SPT-Aki *Client*- it should be installed onto it's own copy of Tarkov. See the Installation>Client section of this readme for further info.
 
 ## [Wiki](https://github.com/stayintarkov/StayInTarkov.Client/blob/master/wiki/Home.md)
 **The Wiki is has been constructed by various contributors. All instructions are also kept within the source in the wiki directory.**
   - ### [Setup Manuals](https://github.com/stayintarkov/StayInTarkov.Client/wiki/Guides-English)
   - ### [FAQs](https://github.com/stayintarkov/StayInTarkov.Client/wiki/FAQs-English)
 
-## Coop
 
-### PREREQUISITE
-You must have the [SPT-Aki mod](https://github.com/stayintarkov/SIT.Aki-Server-Mod) installed in your Server for this module to work. If you do not wish to use the Coop module, you must disable it in the BepInEx config file.
+## Installation
+
+### Overview
+SIT is comprised of 2 major pieces and a launcher
+- A [modified SPT-Aki Server](https://github.com/stayintarkov/SIT.Aki-Server-Mod)
+- The SIT Client Module (this repo!), installed onto an instance of Tarkov
+- [SIT Manager](https://github.com/stayintarkov/SIT.Manager) or [SIT Launcher Classic](https://github.com/stayintarkov/SIT.Launcher.Classic)
+  - You should use the SIT Manager. The classic launcher is only mentioned to clarify for existing classic launcher users.
+
+It's recommended to create the following directory structure to store your SIT Installation. This structure will be referred to for the following sections.
+```
+SIT/
+├── server/      # SPT-Aki Server Mod
+├── game/        # EFT Client
+└── launcher/    # SIT Manager or Classic Launcher
+```
+
+### Server Install
+- Follow the instructions in the SIT SPT-Aki Server Mod repo (linked above) to install and configure the server into the `SIT/server` folder.
+- Exactly *one* person needs to run the server for Coop. This person will need to port forward, or your group will have to connect via Hamachi or some other VPN solution. If you don't know how to do these things, you might see if someone in the SIT discord is willing to help.
+  - In vanilla SPT, you're probably used to running your own local server, and then launching your client which connects to that server under the hood. With SIT, one person will run the modded server and everyone else will connect to that server over the internet.
+
+### Launcher Install
+- Follow the instructions in the SIT Manager repo. Install into the `SIT/launcher` folder.
+
+### Client Install
+- **Everyone** must install the SIT Client Mod. You can install it using SIT Manager, or manually if desired.
+- **IF YOU USE SPT ALREADY**: Do **NOT** install the SIT Client mod onto your existing SPT install. The SIT Client Mod is currently not compatible with the SPT-Aki client, so it needs to be installed on it's own copy of Tarkov.
```

Comment:
"currently not compatible" , you can drop the currently here since probably never be (assembly difference , dll loading difference )

Reply:
Gotcha- I added the qualifier due to the "Why don't you use Aki Module DLLs" sxn in the FAQs.
> The end-goal would be to rely on SPT-Aki and for this to be solely focused on SIT only features.

Does that still stand, or is that comment a bit dated?

Reply:
I think that's okay

Reply:
For clarity, drop the "currently" & keep the faq statement as it is?